EpsilonDriverControls/make.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+if [ "$1" = "clean" ]
+then
+    (cd ../EpsilonDriverControls/ && make clean)
+    (cd Tests/ && make clean)
+    exit
+fi
+
+(cd ../ && ./format.sh EpsilonDriverControls)
+(make -j4 && cd ../EpsilonDriverControls/ && make)
```

Comment:
Same here, moving `../` when formatting doesn't change where you are because that is within parenthesis, so you don't need to `cd` anywhere

Reply:
Ohhh ok I think I missed read something

Reply:
lool wait, this line

1. makes the driver controls code
2. cd's out and back into the driver controls code
3. makes the driver controls code again

Then the next line makes the driver controls again before making the test